deploy/helm/sumologic/README.md

```diff
@@ -482,5 +482,25 @@ The following table lists the configurable parameters of the Sumo Logic chart an
+| `debug.logs.collector.print`                                                          | Set to `true` to print logs processed by log collector on standard output.                                                                                                                                                                                                                                 | `false`                                                                                                                                                                              |
+| `debug.logs.collector.stopLogsIngestion`                                              | Set to `true` to prevents sending log collector logs to Sumo Logic. It prevents incresing ingestion and costs in case of huge amount of log collector logs.                                                                                                                                                | `false`                                                                                                                                                                              |
+| `debug.logs.metadata.forwardToSumologicMock`                                          | Set to `true` to forward logs processed by metadata statefulset also to Sumo Logic Mock.                                                                                                                                                                                                                   | `false`                                                                                                                                                                              |
+| `debug.logs.metadata.print`                                                           | Set to `true` to print logs processed by log metadata pods on standard output.                                                                                                                                                                                                                             | `false`                                                                                                                                                                              |
+| `debug.logs.metadata.stopLogsIngestion`                                               | Set to `true` to prevents sending log metadata logs to Sumo Logic. It prevents incresing ingestion and costs in case of huge amount of log metadata logs.                                                                                                                                                  | `false`                                                                                                                                                                              |
```

Comment:
```suggestion
| `debug.logs.metadata.stopLogsIngestion`                                               | Set to `true` to prevent sending log metadata logs to Sumo Logic. It prevents increasing ingestion and costs in case of huge amount of log metadata logs.                                                                                                                                                  | `false`                                                                                                                                                                              |
```